pq-crypto/EVP_pq_constants.h

```diff
@@ -0,0 +1,38 @@
+//
+// Created by lamjyoti on 8/11/2021.
+//
+
+#ifndef AWSLC_EVP_PQ_CONSTANTS_H
+#define AWSLC_EVP_PQ_CONSTANTS_H
+
+
+/* TLDR s2n: 0=true, -1=false,
+ * TLDR aws-lc: 1=true, 0=false
+ * current work around to keep following macros */
+/* A value which indicates the outcome of a function */
+#define EVP_PQ_SUCCESS 1
+#define EVP_PQ_FAILURE 0
+
+#define IN /* Indicates a necessary function input */
+#define OUT /* Indicates a function output */
+
+
+// Proposed work around for POSIX_GUARD and POSIX_GUARD_RESULT:
+// Currently simplified to only two macros 1 (pass) and 0 (fail)
+#define POSIX_GUARD(result) __EVP_ENSURE((result) == EVP_PQ_SUCCESS, return EVP_PQ_FAILURE)
```

Comment:
I'm still thinking about this. Does aws-lc follow the pattern of returning SUCCESS/FAILURE? For example, when you use `POSIX_GUARD` on `RAND_bytes()` and `random_mod_order_B()`, are the return values consistent with what this macro expects?